docs/source/services.md

```diff
@@ -301,10 +338,10 @@ and taking note of the following process:
 3. On success, the reply will be a JSON model describing the user:
 
    ```json
-   { 
+   {
      "name": "inara",
      "groups": ["serenity", "guild"],
-     
+
    }
    ```
 
```

Comment:
Thank you for adding it! 🍰  + 🍰 

One clarification: nbviewer can be run either as a hub managed or externally managed service. The README for nbviewer does say it. Not sure if it's worth clarifying in the Hub docs here.